### ISSUE TITLE
System db fault in pg connection driver when use group/by

### ISSUE THREAD

**Issue by majioa** [headius/jruby-pg#33]
I have the following error when issuing group by operation in pg connection:

    > self.all.group(:inn)
    => (pry) output error: java.lang.NoSuchMethodError: org.jruby.RubyEncoding.newEncoding(Lorg/jruby/Ruby;Lorg/jcodings/Encoding;)Lorg/jruby/RubyEncoding;

and also I've tried it with plain sql call with the same result:

    > self.connection.execute( "SELECT \"organizations\".* FROM \"organizations\" GROUP BY \"organizations\".\"inn\"")
    Java::JavaLang::NoSuchMethodError: org.jruby.RubyEncoding.newEncoding(Lorg/jruby/Ruby;Lorg/jcodings/Encoding;)Lorg/jruby/RubyEncoding;
    from org.jruby.pg.Connection.newPgErrorCommon(Connection.java:1434)

**Comment by qnet-herwin** [comment]
I guess this has been fixed with commit 9cc73de31ed69aa477928cad86399df0362729f5. Maybe @majioa could confirm this, otherwise it would be hard to reproduce this with this amount of information.

**Comment by majioa** [comment]
@qnet-herwin i can't reproduce or confirm it for now, sorry, because currently the project doesn't use pg-jruby

**Comment by elcubo** [comment]
I just got this error with pg-jruby 0.17.1.

**Comment by donv** [comment]
I can confirm this bug has been fixed in jruby-pg 0.1

**Comment by majioa** [comment]
@donv 0.1?

**Comment by donv** [comment]
Yes @majioa , this gem has changed its name from `pg-jruby` to `jruby-pg` and started the versioning from 0.1.
